test/built-ins/JSON/stringify/string-escape-ascii.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2018 Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-quotejsonstring
+description: >
+    JSON.stringify property names and values containing ASCII
+    characters that require escaping
+---*/
+
+var char_to_json = {
+  '"': '\\"',
+  "\\": "\\\\",
+  "\x00": "\\u0000",
```

Comment:
`\x00` _could_ be `\0` (it’s _not_ an octal escape sequence)

Reply:
I addressed the grammar nits, but left this `\x00` for internal consistency with the following properties.